cypress/support/pages/article.pageObject.js

```diff
@@ -0,0 +1,79 @@
+import PageObject from "../PageObject";
+
+class ArticlePageObject extends PageObject {
+  url = '#/editor';
+
+  get articleTitleField() {
+    return cy.getByDataCy('article-title-field');
+  }
+
+  typeArticleTitle(articleTitle) {
+    this.articleTitleField.clear().type(articleTitle);
+  }
+
+  get articleDescField() {
+    return cy.getByDataCy('article-desc-field');
+  }
+
+  typeArticleDesc(articleDesc) {
+    this.articleDescField.clear().type(articleDesc);
+  }
+
+  get articleBodyField() {
+    return cy.getByDataCy('article-body-field');
+  }
+
+  typeArticleBody(articleBody) {
+    this.articleBodyField.clear().type(articleBody);
+  }
+
+  get articleTagsField() {
+    return cy.getByDataCy('article-tags-field');
+  }
+
+  typeArticleTags(articleTag) {
+    this.articleTagField.clear().type(`${articleTag}{enter}`);
+  }
+
+  get publishBtn() {
+    return cy.getByDataCy('article-publish-btn');
+  }
+
+  clickPublishBtn() {
+    this.publishBtn.click();
+  }
+
+  get container() {
+    return cy.get('.container');
+  }
+
+  verifyAricleCreation(articleTitle) {
```

Comment:
There is a typo in the method name `verifyAricleCreation`. It should be `verifyArticleCreation`.